static/templates/settings/organization_profile_admin.hbs

```diff
@@ -84,11 +84,11 @@
         </div>
         <h3 class="light">
             {{t "Deactivate organization" }}
-            <i class="fa fa-info-circle settings-info-icon realm_deactivation_tooltip tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Only owners can deactivate the organization.' }}"></i>
+            {{> ../help_link_widget link="/help/deactivate-your-organization" }}
         </h3>
         <div class="deactivate-realm-section">
             <div class="input-group">
-                <button class="button rounded btn-danger" id="deactivate_realm_button">
+                <button class="button rounded btn-danger deactivate" id="deactivate_realm_button" title="{{t 'Only organization owners may deactivate an organization.'}}">
```

Comment:
What does the 'deactivate' CSS class here do?

Reply:
Hi! I took inspiration from the file `/static/templates/settings/admin_user_list.hbs` where I found the 'deactivate' CSS class. Maybe it should not be implemented in this case?

`            <button title="{{t 'Deactivate' }}" class="button rounded small deactivate btn-danger" {{#if cannot_deactivate}}disabled="disabled"{{/if}}>
                <i class="fa fa-user-times" aria-hidden="true"></i>
            </button>
`